<launch>
    <rosparam file="$(find yolov5_tensorrtx)/config/model.yaml" command="load" />
    <node pkg="yolov5_tensorrtx" type="yolov5" name="yolov5_tensorrtx" respawn="true" output="screen" />
</launch>